rest/config_flags.go

```diff
@@ -72,6 +72,7 @@ func registerConfigFlags(config *StartupConfig, fs *flag.FlagSet) map[string]con
 		"logging.console.rotation.max_age":                 {&config.Logging.Console.Rotation.MaxAge, fs.Int("logging.console.rotation.max_age", 0, "")},
 		"logging.console.rotation.localtime":               {&config.Logging.Console.Rotation.LocalTime, fs.Bool("logging.console.rotation.localtime", false, "")},
 		"logging.console.rotation.rotated_logs_size_limit": {&config.Logging.Console.Rotation.RotatedLogsSizeLimit, fs.Int("logging.console.rotation.rotated_logs_size_limit", 0, "")},
+		"logging.console.rotation.rotation_interval":       {&config.Logging.Console.Rotation.RotationInterval, fs.String("logging.console.rotation.rotation_interval", "", "")},
```

Comment:
Is it worth adding some tests in `TestFillConfigWithFlagsValidVals` that the default will end up being 0 and this value is correctly populated?

Reply:
Default will be `nil`, but yes I can add some unset values to assert on in there.